alliander/Issues/connection/description.xml

```diff
@@ -0,0 +1,50 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Electricity/Gas connection</en>
+		<nl>Demo Aansluiting</nl>
+	</Name>
+	<ShortName>
+		<en>Demo Connection</en>
+		<nl>Demo Aansluiting</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>alliander</IssuerID>
+	<CredentialID>connection</CredentialID>
+	<Description>
+		<en>Identification numbers (EAN) for your electricity and gas connection. See https://www.eancodeboek.nl/</en>
+		<nl>Identificatienummers (EAN) voor uw electriciteits- en gasaansluiting. See https://www.eancodeboek.nl/</nl>
+	</Description>
+
+	<Attributes>
+		<Attribute id="ean-gas">
+			<Name>
+				<en>EAN Gas</en>
+				<nl>EAN Gas</nl>
+			</Name>
+			<Description>
+				<en>ean code of your gas connection</en>
+				<nl>eancode van uw electriciteitsaansluiting</nl>
```

Comment:
Capitalization of `ean` is (here and in the other attribute descriptions) not consistent with `EAN` elsewhere.